jekyll/_cci2/enable-checks.md

```diff
@@ -4,14 +4,25 @@ title: "Enabling GitHub Checks"
 short-title: "Enabling GitHub Checks"
 description: "How to enable GitHub Checks for CircleCI"
 categories: [getting-started]
-order: 1
-contentTags: 
+contentTags:
   platform:
   - Cloud
 ---
 
+## Introduction
+{: #introduction }
+
 This document describes how to enable the GitHub Checks feature and authorize CircleCI to report workflow status to the GitHub app. **The GitHub checks integration feature is not currently available on CircleCI server**.
 
+GitHub Checks provides you with workflow status messages and gives the option to rerun workflows from the GitHub Checks page.
+
+After GitHub Checks is enabled, CircleCI workflow status is reported under the checks tab on GitHub.
+
+![CircleCI checks on GitHub]({{site.baseurl}}/assets/img/docs/checks_tab.png)
+
+GitHub does not currently provide a granular way for you to rerun workflows. Because CircleCI uses checks that are mapped to workflows (e.g. a single configuration may have one or more workflows), when you select the Re-run checks button, you will automatically re-run all checks, regardless of whether you selected "re-run failed checks" or "rerun all checks" from the Re-run checks button.
```

Comment:
I actually feel like this whole bit can be removed and it'll be easier to read? " Because CircleCI uses checks that are mapped to workflows (e.g. a single configuration may have one or more workflows)"